server/src/db/serializers/DogParksSerializer.js

```diff
@@ -10,6 +10,34 @@ class DogParksSerializer {
         })
         return newParks
     }
-}
+
+    static showDetailsForShow(dogPark) {
+        const allowedAttributes = ["id", "name", "address", "description", "updatedAt"]
+
+        let newPark = {}
+        newPark.tags = []
+        for (const attribute of allowedAttributes) {
+            newPark[attribute] = dogPark[attribute]
+        }
+
+        if (dogPark.hasTrash) {
+            newPark.tags.push("Trash Cans")
+        }
+        if (dogPark.hasBenches) {
+            newPark.tags.push("Benches")
+        }
+        if (dogPark.hasWater) {
+            newPark.tags.push("Water Fountains")
+        }
+        if (dogPark.hasFence) {
+            newPark.tags.push("Fencing")
+        }
+        if (dogPark.hasBags) {
+            newPark.tags.push("Doggie Bags")
+        }
```

Comment:
ahhh, okay. Rather than add these tags here, you should add them as a constant in react or in DogPark.js 

If you add them to the model, then every new DogPark should have a property you can call on:
e.g. 

```
class DogPark extends Model {
     
  get tags(){
    return ["all dogs", "small-dogs", "big-dogs"]
  }
}
```
then in the serializer you can just say 
`newPark.tags = dogPark.tags`

Reply:
alternatively, you can just add this array to your show page component and use it there!

regardless, the _serializer_ shouldnt be the one adding those tags. The serializer just chooses what to keep and what not to